tangelo/algorithms/projective/tests/test_iqpe.py

```diff
@@ -120,8 +122,8 @@ def test_qubit_hamiltonian_input(self):
         # Test that get_resources returns expected results before running the simulation.
         resources = iqpe.get_resources()
         self.assertEqual(resources["applied_circuit_width"], 5)
-        self.assertEqual(resources["applied_circuit_depth"], 1999, f"{iqpe.circuit.applied_gates}")
-        self.assertEqual(resources["applied_circuit_2qubit_gates"], 1372)
+        self.assertEqual(resources["applied_circuit_depth"], 412, f"{Circuit(generate_applied_gates(iqpe.circuit))}")
```

Comment:
You're gonna print a circuit of potential depth 400 if this fails ? 

Does this really help anyone to find what went wrong ? I'd rather print just the actual depth, and the developer can then debug / print circuit attributes to look into what this circuit is.